src/main/groovy/org/mockito/release/notes/contributors/ContributorsMap.java

```diff
@@ -0,0 +1,8 @@
+package org.mockito.release.notes.contributors;
+
+import org.mockito.release.notes.model.Contributor;
+
+import java.util.Map;
+
+public interface ContributorsMap extends Map<String, Contributor> {
```

Comment:
This public type seems like a maintenance burden in the long term because it inherits lots of public methods for Map interface and it increases public API footprint. How about we add a new interface like:

```java
interface ContributorProvider {
  Contributor findContributor(String authorName);
}
```

Another option is to drop ContributorsMap interface and simply use Map<String, Contributor>.